js/main.js

```diff
@@ -12,11 +12,11 @@ var $backButton = document.querySelector('#previous');
 $gstart.addEventListener('click', changePage);
 
 function changePage(event) {
```

Comment:
This function name doesn't describe the action that is taking place. Which page are you changing to?